spyder/utils/ipython/style.py

```diff
@@ -0,0 +1,215 @@
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Style for IPython Console
+"""
+# Standard imports
+import os
+
+# Local imports
+from spyder.config.gui import get_color_scheme
+from spyder.config.main import CONF
+
+# Third party imports
+from qtconsole.styles import dark_color
+
+
+def create_qss_style():
+    """Returns a QSS stylesheet with Spyder color scheme settings.
+
+    The stylesheet can contain classes for:
+        Qt: QPlainTextEdit, QFrame, QWidget, etc
+       Pygments: .c, .k, .o, etc. (see PygmentsHighlighter)
+        IPython: .error, .in-prompt, .out-prompt, etc
+    """
+
+    def give_font_weight(is_bold):
+        if is_bold:
+            return 'bold'
+        else:
+            return 'normal'
+
+    def give_font_style(is_italic):
+        if is_italic:
+            return 'italic'
+        else:
+            return 'normal'
+
+    selected_color_scheme = CONF.get('color_schemes', 'selected')
+    color_scheme = get_color_scheme(selected_color_scheme)
+    fon_c, fon_fw, fon_fs = color_scheme['normal']
+    font_color =  fon_c
+    if dark_color(font_color):
+        in_prompt_color = 'navy'
+        out_prompt_color = 'darkred'
+    else:
+        in_prompt_color = 'lime'
+        out_prompt_color = 'red'
+    background_color = color_scheme['background']
+    selection_background_color = '#ccc'
+    error_color = 'red'
+    in_prompt_number_font_weight = 'bold'
+    out_prompt_number_font_weight = 'bold'
+    inverted_background_color = font_color
+    inverted_font_color = background_color
+
+    sheet = """QPlainTextEdit, QTextEdit, ControlWidget {{
+                                          color: {} ;
+                                          background-color: {};
+                                          selection-background-color: {}
+                                         }}
+              .error {{ color: {}; }}
+              .in-prompt {{ color: {}; }}
+              .in-prompt-number {{ color: {}; font-weight: {}; }}
+              .out-prompt {{ color: {}; }}
+              .out-prompt-number {{ color: {}; font-weight: {}; }}
+              .inverted {{ color: {}; background-color: {}; }}
+              """
+
+    sheet_formatted = sheet.format(font_color, background_color,
+                                   selection_background_color,
+                                   error_color,
+                                   in_prompt_color, in_prompt_color,
+                                   in_prompt_number_font_weight,
+                                   out_prompt_color, out_prompt_color,
+                                   out_prompt_number_font_weight,
+                                   inverted_background_color,
+                                   inverted_font_color)
+
+    return sheet_formatted
+
+def get_syntax_style_dictionary():
+    """Create a dictionary to create the syntax style."""
```

Comment:
Change this to 

    """
    Create a dictionary that saves the current Spyder color scheme as a
    Pygments style.
    """